content/docs/user-guide/testing/quality/_index.md

```diff
@@ -0,0 +1,216 @@
+---
+description: Use the Quality system to define quality levels and the console variable settings for each level.
+title: Quality System
+linkTitle: Quality System
+---
+
+The Quality system gives you the ability to define quality levels and console variable settings for each quality level. Applications typically run on different hardware and need different settings to achieve the best performance on each device and accomodate user preferences.
+
+
+## Quality groups
+
+Quality groups (e.g. `q_general`, `q_graphics`, `q_physics` etc.) can be defined in [Settings Registry](/docs/user-guide/settings/) `.setreg` files to make it easier to group similar settings in your application.  Quality levels (e.g. `Low`, `Medium`, `High`, `VeryHigh`) can be defined within each group to accomodate the needs of your application. 
+
+The Quality system creates console variables on demand for each quality group so you can change the active quality level for that group at runtime using the console, for this reason we recommend that you use a lowercase group name with a `q_` prefix, which is short for `quality`.  For example, you may define a quality group for your game named `q_game` and set the quality level at runtime using the console command like the one below where `<quality level>` is the number index or text name of the quality level.
+```
+q_game <quality level>
+```
+
+Quality group objects should be defined at the path `O3DE/Quality/Groups` and have the following format.
+
+```json
+{
+    "O3DE": {
+        "Quality": {
+            "Groups": {
+                "q_general": {
+                    "Description" : "Description with values e.g.. 0 : Low, 1 : Medium, 2 : High",
+                    "Levels": [ // quality level names
+                        "Low",
+                        "Medium",
+                        "High",
+                        ...
+                    ],
+                    "Default": 3, // default quality level index in the Levels array
```

Comment:
We should probably change this example quality level settings Default value to be an index that is within this group range of 0-2, since this description only goes up to "HIgh".
```suggestion
                    "Description" : "Description with values e.g.. 0 : Low, 1 : Medium, 2 : High",
                    "Levels": [ // quality level names
                        "Low",
                        "Medium",
                        "High",
                        ...
                    ],
                    "Default": 1, // default quality level index in the Levels array
```